action-templates/actions/action-npm-release/action.yml

```diff
@@ -1,35 +1,28 @@
-name: Npm Release
+name: "Npm Release"
 
 inputs:
   node-version:
+    description: "node version"
     required: false
     default: "22"
-    type: string
-    description: node version
   app-path:
+    description: "path where the package.json is located"
     required: true
-    type: string
-    description: path where the package.json is located
   releaseVersion:
-    type: choice
-    description: Select version increment type (follows Semantic Versioning)
+    description: "Select version increment type (follows Semantic Versioning)"
     required: true
-    options:
```

Comment:
ich finde das mit der Auswahlliste ganz gut. Natürlich gibt es noch weiter Optionen https://docs.npmjs.com/cli/v8/commands/npm-version#synopsis
Schade, dass es bei der Auswahllist kein Sonstiges Feld gibt, wo man selber etwas reinschreiben kann

Reply:
Bin ich deiner Meinung, nachdem es aber bei Actions keinen type gibt, bringt das hier nix, das muss bzw. ist im Workflow so definiert.

Reply:
ach stimmt ist hier drin https://github.com/it-at-m/refarch-templates/blob/main/.github/workflows/npm-release.yml#L10